services/catarse/db/migrate/20191126180030_add_is_adult_content_to_project_details_view.rb

```diff
@@ -0,0 +1,220 @@
+       state_order(p.*) AS state_order,
+       p.expires_at,
+       zone_timestamp(p.expires_at) AS zone_expires_at,
+       online_at(p.*) AS online_date,
+       zone_timestamp(online_at(p.*)) AS zone_online_date,
+       zone_timestamp(in_analysis_at(p.*)) AS sent_to_analysis_at,
+       is_published(p.*) AS is_published,
+       is_expired(p.*) AS is_expired,
+       open_for_contributions(p.*) AS open_for_contributions,
+       p.online_days,
+       remaining_time_json(p.*) AS remaining_time,
+       elapsed_time_json(p.*) AS elapsed_time,
+       posts_size.count AS posts_count,
+       json_build_object('city', ct.name, 'state_acronym', st.acronym, 'state', st.name) AS address,
+       json_build_object('id', u.id, 'name', u.name, 'public_name', u.public_name) AS "user",
+       ( SELECT count(DISTINCT pr_1.user_id) AS count
+              FROM project_reminders pr_1
+             WHERE (pr_1.project_id = p.id)) AS reminder_count,
+       is_owner_or_admin(p.user_id) AS is_owner_or_admin,
+       user_signed_in() AS user_signed_in,
+       current_user_already_in_reminder(p.*) AS in_reminder,
+       posts_size.count AS total_posts,
+       (((p.state)::text = 'successful'::text) AND ((p.expires_at)::date >= '2016-06-06'::date)) AS can_request_transfer,
+       ("current_user"() = 'admin'::name) AS is_admin_role,
+       (EXISTS ( SELECT true AS bool
+              FROM (contributions c_1
+                JOIN user_follows uf ON ((uf.follow_id = c_1.user_id)))
+             WHERE ((is_confirmed(c_1.*) AND (uf.user_id = current_user_id())) AND (c_1.project_id = p.id)))) AS contributed_by_friends,
+           CASE
+               WHEN ("current_user"() = 'admin'::name) THEN NULLIF(btrim((array_agg(DISTINCT admin_tags_lateral.tag_list))::text, '{}'::text), 'NULL'::text)
+               ELSE NULL::text
+           END AS admin_tag_list,
+       NULLIF(btrim((array_agg(DISTINCT tags_lateral.tag_list))::text, '{}'::text), 'NULL'::text) AS tag_list,
+       ct.id AS city_id,
+           CASE
+               WHEN ("current_user"() = 'admin'::name) THEN p.admin_notes
+               ELSE NULL::text
+           END AS admin_notes,
+           CASE
+               WHEN ("current_user"() = 'admin'::name) THEN p.service_fee
+               ELSE NULL::numeric
+           END AS service_fee,
+       has_cancelation_request(p.*) AS has_cancelation_request,
+           CASE
+               WHEN is_owner_or_admin(p.user_id) THEN can_cancel(p.*)
+               ELSE false
+           END AS can_cancel,
+       p.tracker_snippet_html,
+       cover_image_thumb(p.*) AS cover_image,
+       p.common_id,
+       CASE
+            WHEN p.content_rating >= 18 THEN true
+            ELSE false
+        END AS is_adult_content,
+       p.content_rating
+      FROM ((((((((projects p
+        JOIN categories c ON ((c.id = p.category_id)))
+        JOIN users u ON ((u.id = p.user_id)))
+        LEFT JOIN project_totals pt ON ((pt.project_id = p.id)))
```

Comment:
we should use "1".project_totals instead project_totals

Reply:
done